### ISSUE TITLE
`content.tooltips` feature breaks abbreviation hover in tab titles

### ISSUE THREAD

**Issue by SerTetora** [squidfunk/mkdocs-material#6488]
### Bug description

I'm trying to use improved tooltips within content tabs.

**with:** 
```
features:
    - content.tooltips
``` 
![image](https://github.com/squidfunk/mkdocs-material/assets/95703687/9bf39a36-4603-4f32-a005-097b30462852)
on scroll
![image](https://github.com/squidfunk/mkdocs-material/assets/95703687/53c8650b-77a1-458b-86c5-4785cfa9d94a)

**without:**
![image](https://github.com/squidfunk/mkdocs-material/assets/95703687/6cb49970-c81d-49ff-bad3-fa7f73a7ead3)

<details><summary>mkdocs.yml</summary>
<p>

```
site_name: My Docs
docs_dir: docs/

nav: 
  - index.md

theme:
  name: material
  features:
    - content.tooltips

plugins:
  - info

markdown_extensions:
  - abbr
  - attr_list
  - pymdownx.snippets:
      auto_append:
        - docs/includes/abbreviations.md
  - pymdownx.superfences
  - pymdownx.tabbed:
      alternate_style: true
```

</p>
</details> 

<details><summary>docs/index.md</summary>
<p>

```
# Welcome to MkDocs

=== "FEM"
    lorem FEM

=== "CFD"
    ipsum CFD
```

</p>
</details> 

<details><summary>docs/includes/abbreviations.md</summary>
<p>

```
*[FEM]: Finite Element Method
*[CFD]: Computational Fluid Dynamic

``` 

</p>
</details> 

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- [Content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/)
- [Improved tooltips](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#improved-tooltips)
- [Adding a glossary](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#adding-a-glossary)

### Reproduction

[9.5.1-tooltips_reproduction.zip](https://github.com/squidfunk/mkdocs-material/files/13632949/9.5.1-tooltips_reproduction.zip)

### Steps to reproduce

Use improved tooltips within content tabs.

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by Andre601** [comment]
I am not sure if this is related, but I encounter a position issue with tooltips.

Namely, when I use inline admonition blocks (`inline end` type) and a text with abbreviation is split up by it (or otherwise moved by it) does the tooltip not recognize it and display it at a completely wrong position.

My minimal reproduction:
[9.5.5-tooltips-position-issue.zip](https://github.com/squidfunk/mkdocs-material/files/14054588/9.5.5-tooltips-position-issue.zip)

**Comment by squidfunk** [comment]
@SerTetora thanks for reporting. We've just implemented a new tooltip positioning logic for the brand new [footnote tooltips](https://squidfunk.github.io/mkdocs-material/reference/footnotes/#footnote-tooltips) ([see demo on Twitter](https://twitter.com/squidfunk/status/1750082128921162113)), which does not suffer from the problems reported. We're in the process of replacing the previous tooltip positioning with the new implementation, which should also solve this issues as well.

**Comment by squidfunk** [comment]
Note that this problem is already fixed in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/), as we swapped out the tooltip implementation to implement [footnote tooltips](https://squidfunk.github.io/mkdocs-material/reference/footnotes/#footnote-tooltips) and [instant previews](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#instant-previews). Once we've collected a little more feedback on those features, we'll back port the new business logic which we confirmed to also fix this problem ☺️

**Comment by squidfunk** [comment]
Fixed in e15518cfa588d3dfb8113c1ba90f2c0bc5c02fce – I backported the new tooltip implementation from Insiders in the [refactor/tooltip-positioning](https://github.com/squidfunk/mkdocs-material/tree/refactor/tooltip-positioning) branch! If you could spare a few minutes and check whether this fixes your issue, we can release it in the next days ☺️ Install with:

```
pip install git+https://github.com/squidfunk/mkdocs-material.git@refactor/tooltip-positioning
```

**Comment by SerTetora** [comment]
Hi, @squidfunk, it works for me. Many thanks for fixing this!

**Comment by squidfunk** [comment]
Awesome, thanks for testing!

**Comment by Andre601** [comment]
Can also confirm that it seems fixed now.

**Comment by squidfunk** [comment]
Released as part of [9.5.18](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.18).
